src/modules/endpoint/models/CategoryModel.js

```diff
@@ -0,0 +1,65 @@
+class CategoryModel {
+  constructor ({ id, url = '', title = '', parent = 0, content = '', thumbnail = null, order = 0, availableLanguages = {} }) {
+    this._id = id
+    this._url = url
+    this._title = title
+    this._content = content
+    this._parent = parent
+    this._thumbnail = thumbnail
+    this._order = order
+    this._children = []
+    this._availableLanguages = availableLanguages
+  }
+
+  get thumbnail () {
+    return this._thumbnail
+  }
+
+  addChild (id) {
+    this._children.push(id)
+    this._children = this._children.sort((category1, category2) => category1.order - category2.order)
+  }
+
+  get id () {
+    return this._id
+  }
+
+  get url () {
+    return this._url
+  }
+
+  get title () {
+    return this._title
+  }
+
+  get content () {
+    return this._content
+  }
+
+  get parent () {
+    return this._parent
+  }
+
+  get children () {
+    return this._children
+  }
```

Comment:
In my opinion we don't need children as a property of a CategoryModel anymore (that would be an optimization) - since we can always replace that by 
```javascript
 allCategores.filter(category => category.parent === myCategory.id)
```

Reply:
you're right, i'll remove it

Reply:
We'll need the children to render a list of children. And we already have those there. So why not use them? We'd actually have to test more code.